sdk/java-sdk-protobuf-testkit/src/main/java/kalix/javasdk/testkit/KalixProxyContainer.java

```diff
@@ -88,7 +88,7 @@ public KalixProxyContainer(
     if ("false".equals(System.getenv("VERSION_CHECK_ON_STARTUP"))) {
       withEnv("VERSION_CHECK_ON_STARTUP", "false");
     }
-    waitingFor(Wait.forLogMessage(".*gRPC proxy started.*", 1));
+    waitingFor(Wait.forLogMessage(".*(gRPC proxy|Kalix Runtime) started.*", 1));
```

Comment:
https://github.com/lightbend/kalix-runtime/pull/2175/files#r1404430716

Reply:
Oh, good catch. I completely forgot about this. 

Can good trick to match on both. 👍

Reply:
We put the old log output back, so that bumping the Runtime works even if the Testkit isn't upgraded.
- https://github.com/lightbend/kalix-runtime/pull/2238